inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -424,7 +424,7 @@ GnaWaitStatus GNADeviceHelper::wait(uint32_t reqId, int64_t millisTimeout) {
     if (status == Gna2StatusDriverQoSTimeoutExceeded) {
         return GNA_REQUEST_ABORTED;
     }
-    checkGna2Status(status, "Gna2RequestWait");
+    //checkGna2Status(status, "Gna2RequestWait");
```

Comment:
@nkogteva I have a question regarding this PR (addressing an idea to recover when observing low level e.g., GNA Lib/Drv issues).

When GNA Plugin get some abnormal status from `Gna2RequestWait()` now it is throwing exception.
OV IE User code can capture this exception (around `inferRequest.Wait()`) and read some extra information from the message (e.g. what was the original status from `Gna2RequestWait()`).

Should we left as is (i.e. throw exception) or return some IE code for this e.g. `InferenceEngine::GENERAL_ERROR`?

If plugin resigns throwing exception could it pass extra information somehow anyway (i.e., when returning `InferenceEngine::GENERAL_ERROR`). Is there some IE API for this (Release build)?

Reply:
@apankratovantonp could you please advice?